src/client.ts

```diff
@@ -158,6 +158,23 @@ export class PluggyClient extends BaseApi {
     )
   }
 
+  /**
+   * Post transaction user category for transactin
+   * @param id The Transaction id
+   *
+   * @returns {Transaction} updated transaction object
+   */
+  async patchTransactionCategory(id: string, categoryId: string): Promise<Transaction> {
```

Comment:
I don't think the name of this method is consistent with the rest of the project, that'd be something like `updateTransactionCategory` and not patch

Reply:
Can we also add the usage of this method in the example app? so at least we have a test-ish of this part

Reply:
for sake of simplicity and have it all up to date, could we merge, wait for semantic release and use latest version on the example app on a new pr?